RickAndMortyFunApp/NetworkServices/NetworkService.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  NetworkService.swift
+//  RickAndMortyFunApp
+//
+//  Created by Rusłan Chamski on 26/02/2019.
+//  Copyright © 2019 Rusłan Chamski. All rights reserved.
+//
+
+import Foundation
+
+class NetworkService {
+    
+    func loadCharcters(compltion: @escaping([Character]) -> Void) {
+        let urlString = "https://rickandmortyapi.com/api/character/"
+        guard let url = URL(string: urlString) else { return }
+
+
+        URLSession.shared.dataTask(with: url) { (data, respone, error) in
+            guard let data = data else { return }
+
+            do {
+               let characters = try JSONDecoder().decode(Characters.self, from: data)
+                DispatchQueue.main.async {
+                    compltion(characters.results)
+                }
+            } catch {
+                print("error: \(error)")
```

Comment:
Here you should also call completion. Not sure, but definitely better to call:
```
DispatchQueue.main.async {
    completion([])
}
```
This way we at least know that we got response